### PR TITLE
Made pageview servlet

### PR DESCRIPTION
**Pull request by Dhindhu**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DateServlet.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that responds with the current date. */
+@WebServlet("/date")
+public class DateServlet extends HttpServlet {
+
+    /**
+     *
+     */
```
Comment (zsxking) [inline portfolio/src/main/java/com/google/sps/servlets/DateServlet.java]: empty comments?

File: portfolio/src/main/java/com/google/sps/servlets/DateServlet.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    private static final long serialVersionUID = 1L;
+
+    @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (zsxking) [inline portfolio/src/main/java/com/google/sps/servlets/DateServlet.java]: fix aligment

**Review by zsxking — APPROVED** [review]
couple nit. otherwise LGTM

File: portfolio/src/main/java/com/google/sps/servlets/DateServlet.java (hunk 1 above)
Comment (Dhindhu) [reply portfolio/src/main/java/com/google/sps/servlets/DateServlet.java]: Removed the empty comments

File: portfolio/src/main/java/com/google/sps/servlets/DateServlet.java (hunk 2 above)
Comment (Dhindhu) [reply portfolio/src/main/java/com/google/sps/servlets/DateServlet.java]: Done, thanks!

**Pull request merged**